resources/puppetlabs/lein-ezbake/template/pe/ext/redhat/init.erb

```diff
@@ -75,20 +72,13 @@ start() {
     <% end -%>
 
     pushd "${INSTALL_DIR}" &> /dev/null
-    daemon --user $USER --pidfile $PIDFILE "$EXEC >> /var/log/puppetlabs/<%= EZBake::Config[:real_name] %>/<%= EZBake::Config[:real_name] %>-daemon.log 2>&1 &"
-    retval=$?
-    sleep 1
+    daemon --user $USER --pidfile $PIDFILE "/opt/puppetlabs/server/apps/<%= EZBake::Config[:real_name] %>/bin/<%= EZBake::Config[:real_name] %> start"
     popd &> /dev/null
-    find_my_pid
+    retval=$?
```

Comment:
This should be above the popd, otherwise it will capture the return of the popd call.